python/dgl/heterograph.py

```diff
@@ -221,6 +221,10 @@ def add_nodes(self, num, data=None, ntype=None):
         * If the key of ``data`` contains new feature fields, those features for
         the old nodes will be created by initializers defined with
         :func:`set_n_initializer` (default initializer fills zeros).
+        *This function discards the batch information. Please use
+        :func:`dgl.DGLGraph.set_batch_num_nodes`
+        and :func:`dgl.DGLGraph.set_batch_num_edges` on the transformed graph
+        to maintain the information.
```

Comment:
Seem to miss a white space after '*'. Does the built doc looks good?

Reply:
Good catch! Fixed.